src/components/CardJob.jsx

```diff
@@ -0,0 +1,53 @@
+import { baseUrl } from '../constants';
+import './CardJob.css';
+import { ButtonLanguage } from './ButtonsLanguages';
+
+function CardJob(props) {
+  const {
+    company,
+    logo,
+    position,
+    postedAt,
+    contract,
+    location,
+    featured,
+    languages,
+    role,
+    level,
+  } = props;
+
+  const logoBase = baseUrl + logo.slice(1);
```

Comment:
Mate, you added images and data.json twice. Can we have only one each?

Reply:
Sorry, I don't understand what you mean

Reply:
We have `/public/images` and `src/assets/images` folders. We don't need both.
The same for `data.json`.